packages/code-du-travail-modeles/src/modeles/conventions/1702_ouvriers_travaux_public/__tests__/rupture-conventionnelle/calculate.spec.ts

```diff
@@ -0,0 +1,54 @@
+import { RuptureConventionnellePublicodes } from "../../../../../publicodes";
+
+const engine = new RuptureConventionnellePublicodes(
+  modelsRuptureConventionnel,
+  "1702"
+);
+
+describe("Un seul type de licenciement pour la CC 1702", () => {
+  test("Missing variables", () => {
+    const input = {
+      "contrat salarié . convention collective": "'IDCC1702'",
+      "contrat salarié . indemnité de licenciement . inaptitude suite à un accident ou maladie professionnelle":
+        "non",
+    };
+
+    const result = engine.calculate(
+      input,
+      "contrat salarié . indemnité de licenciement . résultat conventionnel"
+    );
+    const missingArgs = result.missingArgs.filter((item) => item.rawNode.cdtn);
+    expect(missingArgs.length).toEqual(1);
+    expect(result.missingArgs).toHaveNextMissingRule(
+      "contrat salarié . convention collective . ouvriers travaux public . indemnité de licenciement . age"
+    );
```

Comment:
```suggestion
    const { missingArgs } = engine.calculate(
      input,
      "contrat salarié . indemnité de licenciement . résultat conventionnel"
    );
    expect(result.missingArgs).toHaveNextMissingRule(
      "contrat salarié . convention collective . ouvriers travaux public . indemnité de licenciement . age"
    );
```

on peut encore plus simplifier :)